### PR TITLE
Cg/some commitments in prover

### PR DESCRIPTION
**Pull request by codygunton**
Compute actual commitments in the pre-sumcheck prover rounds.

### CODE REVIEW

File: cpp/src/aztec/polynomials/polynomial.hpp [hunk 1]
```diff
@@ -76,6 +76,8 @@ template <typename Fr> class Polynomial {
 
     Fr* get_coefficients() const { return coefficients_; };
     Fr* get_coefficients() { return coefficients_; };
+    // TODO(Cody): This is a hack.
+    std::span<Fr> get_coefficients_span() { return std::span<Fr>(coefficients_, size()); };
```
Comment (Rumata888) [inline cpp/src/aztec/polynomials/polynomial.hpp]: Wouldn't it be better to define a conversion function?

**Review by Rumata888 — CHANGES_REQUESTED** [review]
Everything seems ok. I'd just make 1 small change

File: cpp/src/aztec/polynomials/polynomial.hpp (hunk 1 above)
Comment (codygunton) [reply cpp/src/aztec/polynomials/polynomial.hpp]: Yes, addressed that as you suggested.

**Review by Rumata888 — APPROVED** [review]
Everything looks good

**Pull request merged**